src/components/har-barn-fortrolig-adresse/HarBarnFortroligAdresse.tsx

```diff
@@ -0,0 +1,24 @@
+import { Tag } from '@navikt/ds-react';
+import { Barn } from '../../types/Personalia';
+import styles from './HarBarnFortroligAdresse.module.css';
+
+interface HarBarnFortroligAdresse {
+    barn: Barn[];
+}
+
+const HarBarnFortroligAdresse = ({ barn }: HarBarnFortroligAdresse) => {
+    const fortroligAdresse = utledFortroligAdresseForBarn(barn);
+    return fortroligAdresse ? (
+        <Tag variant="error" className={styles.personaliaHeader__barnTag}>
+            {fortroligAdresse}
```

Comment:
Her ser det ikke ut som vi tar høyde for at man både kan ha barn som er fortrolige og andre barn som er strengt fortrolige. Altså trenger vi å vise begge varianter av meldingen i det tilfellet? Usikker. 🤔